tests/plugin_tests/test_http_headers_plugin.py

```diff
@@ -48,4 +48,33 @@ def test_hsts_and_hpkp_disabled(self):
 
     def test_hpkp_enabled(self):
         # HPKP is being deprecated in Chrome - I couldn't find a website with the header set
-        pass
\ No newline at end of file
+        pass
+
+    def test_expect_ct_disabled(self):
+      server_info = ServerConnectivityInfo(hostname='hsts.badssl.com')
+      server_info.test_connectivity_to_server()
+
+      plugin = HttpHeadersPlugin()
+      plugin_result = plugin.process_task(server_info, HttpHeadersScanCommand())
+
+      self.assertFalse(plugin_result.expect_ct_header)
+
+      self.assertTrue(plugin_result.as_text())
+      self.assertTrue(plugin_result.as_xml())
+
+      self.assertTrue(pickle.dumps(plugin_result))
+
+    def test_expect_ct_enabled(self):
+      # Github was the only server I could find with expect-ct header set
+      server_info = ServerConnectivityInfo(hostname='github.com')
+      server_info.test_connectivity_to_server()
+
+      plugin = HttpHeadersPlugin()
+      plugin_result = plugin.process_task(server_info, HttpHeadersScanCommand())
+
+      self.assertTrue(plugin_result.expect_ct_header)
```

Comment:
Can you also do some basic checking of the `expect_ct_header`'s attributes (max age, etc.)? Just using `self.assertTrue()` (to make sure there's a value) is probably enough.